### DISCUSSION TITLE
Token Pocket + WC2 failing to sign typed data

### DISCUSSION THREAD

**Discussion by Matko95** [WalletConnect/walletconnect-monorepo#2807] [Developer Support]
Hey guys, getting this error when attempting to sign typed data.

Error: Missing or invalid. request() method: eth_signTypedData

or 

Error: Missing or invalid. request() method: eth_signTypedData_v4

**Comment by finessevanes** [comment]
@Matko95 could you make sure you're using the latest version? This issue was fixed this morning.

**Reply by Matko95** [reply]
Token Pocket still hangs when attempting to sign typed data

**Comment by grahac** [comment]
Looks like this broken again? eth_signTypedData_v4 is giving us errors.
